pages/_app.js

```diff
@@ -13,15 +13,32 @@ export default function App({ Component, pageProps }) {
 
   const [favorites, setFavorites] = useLocalStorageState("favorite", {
     defaultValue: [],
-  }); // to generic
+  });
 
-  const [pairings, setPairings] = useLocalStorageState("parings", {
-    defaultValue: pairingsData,
+  const [pairings, setPairings] = useLocalStorageState("pairings", {
+    defaultValue: pairingsData.map((pairing) => ({
+      ...pairing,
+      rating: 0,
+      totalRatings: 0,
+    })),
   });
+
   const [pairingsInfo, setPairingsInfo] = useLocalStorageState("pairingsInfo", {
     defaultValue: [],
   });
 
+  const [comments, setComments] = useLocalStorageState("comment", {
+    defaultValue: [],
+  });
+
+  useEffect(() => {
+    const storedPairings = JSON.parse(localStorage.getItem("pairings"));
+
+    if (storedPairings) {
+      setPairings(storedPairings);
+    }
+  }, []);
```

Comment:
You already have a localStorage in lines 18-24 that sets and reads from the localStorage with the name "pairings".

This useEffect now gets the data again from localStorage and overwrites the pairings state. You should remove these lines.

```suggestion
```